File: Python/Remove_Duplicates_from_Sorted_Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        x = 1
        for i in range(1,len(nums)):
            if nums[i]  !=  nums[i - 1]:
                nums[x] = nums[i]
                x+=1;     
        return x 

#https://leetcode.com/problems/remove-duplicates-from-sorted-array/?envType=problem-list-v2&envId=niqxdky2

File: CSharp/Remove_Duplicates_from_Sorted_Array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int x = 1 ;  // index if the  next unique elemen
        for(int i = 1; i < nums.Length; i++){
            if(nums[i] != nums[i-1])
            {
                nums[x] = nums[i];
                x++;              ///  if not unique we don't do any changes ,
                                  /// when a unique will  overwrite the element  on index x.
            }
        }
        return x;
    }
}
// https://leetcode.com/problems/remove-duplicates-from-sorted-array/description/?envType=problem-list-v2&envId=niqxdky2
// 26
